my-demo-project/calc/src/test/java/org/example/demo/calc/CalcTest.java

```diff
@@ -0,0 +1,30 @@
+package org.example.demo.calc;
+
+import org.junit.Test;
+
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class CalcTest {
+
+    @Test
+    public void averageMarkOfStudent() {
+        //given
+        List<Integer> integerList=List.of(1,2,3,4,5);
```

Comment:
По код-стайлу нужны разделяющие пробелы. Аналогично и в другом методе этого класса.
```suggestion
        List<Integer> integerList=List.of(1, 2, 3, 4, 5);
```